docker/Dockerfile_spacy_sklearn_duckling

```diff
@@ -0,0 +1,39 @@
+FROM python:3.6-slim
```

Comment:
is this one different from the dockerfile for spacy_sklearn or is it just using a different config file?